query-builder/src/main/java/com/datastax/oss/driver/internal/querybuilder/schema/OptionsUtils.java

```diff
@@ -60,6 +61,8 @@ private static String extractOptionValue(@NonNull Object option) {
             .append(extractOptionValue(subOption.getValue()));
       }
       optionValue.append("}");
+    } else if (option instanceof byte[]) {
+      optionValue.append(ByteUtils.toHexString((byte[]) option));
```

Comment:
Pretty sure we don't want this change in this method; see comments elsewhere in the review for rationale.